node/main.go

```diff
@@ -0,0 +1,15 @@
+package node
+
+import (
+	log "github.com/ChainSafe/log15"
```

Comment:
Wouldn't it be better if the changes were pushed upstream and that was imported?

Reply:
Not sure im following you? 

This is a custom package, just a simple logger.

Reply:
hmm never mind, the original repo that you guys forked off of seems abandoned.

Reply:
ohhh, yeah sorry if that wasn't clear. We also updated it a bit.